### PR TITLE
[release] Update data to 2021_05_04_01_01_15

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: test/integration/golden_response/staging/get_triples/stat_var.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/d7tbsb1"
+      "provenanceId": "dc/kyzlg31"
```
Comment (shifucun) [inline test/integration/golden_response/staging/get_triples/stat_var.json:11]: this provenance change does not seem right

File: test/integration/golden_response/staging/get_stat_set_within_place/US_State.json [hunk 2]
```diff
@@ -388,365 +388,365 @@
...
+          "value": 277417,
           "metadata": {
             "import_name": "BLS_LAUS"
           }
         },
         "geoId/72": {
-          "date": "2020-08",
-          "value": 960458,
+          "date": "2021-03",
+          "value": 989571,
```
Comment (pradh) [inline test/integration/golden_response/staging/get_stat_set_within_place/US_State.json:749]: BLS Update

File: test/integration/golden_response/staging/get_triples/stat_var.json (hunk 1 above)
Comment (pradh) [reply test/integration/golden_response/staging/get_triples/stat_var.json:11]: Yes, not expected... something regressed, I'll take a look.

Comment (pradh) [reply test/integration/golden_response/staging/get_triples/stat_var.json:11]: The root cause is cl/371847183 introduced a new import tagged as CURATED_STATVAR.

But the logic that assigns StatVar provenances assumed there was one CURATED_STATVAR entry only (http://google3/datacommons/import/util/flume_helper.cc?l=528-531&rcl=371903667). I'll fix it now and get it into tonight's build.

Should we defer data rollout to tomorrow?

Comment (shifucun) [reply test/integration/golden_response/staging/get_triples/stat_var.json:11]: defer sg, maybe just release the migration clean up?

Comment (beets) [reply test/integration/golden_response/staging/get_triples/stat_var.json:11]: thanks for catching this -- i'll close the PR and just push the migration cleanup

**Comment by beets** [comment]
closing this PR due to data regression

**Pull request closed without merging**